content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -175,6 +175,66 @@ spec:
 
 However, the particular path specified in the custom recycler Pod template in the `volumes` part is replaced with the particular path of the volume that is being recycled.
 
+### PersistentVolume deletion protection finalizer
+{{< feature-state for_k8s_version="v1.24" state="beta" >}}
+
+Finalizers can be added on a PersistentVolume to ensure that PersistentVolumes
+having `Delete` reclaim policy are deleted only after the backing storage are deleted.
+
+The finalizer `kubernetes.io/pv-controller` is added to in-tree plugin volumes. The following is an example
+
+```shell
+kubectl describe pv task-pv-volume
+Name:            task-pv-volume
+Labels:          type=local
+Annotations:     <none>
+Finalizers:      [kubernetes.io/pv-protection, kubernetes.io/pv-controller]
```

Comment:
`kubernetes.io/pv-controller` is added in 1.24 as beta.
`kubernetes.io/pv-protection` has been there for a while.
I wonder if this gives people impression that both are added in 1.24?  We probably should add a note to clarify.